.github/workflows/testRE.R

```diff
@@ -1,3 +1,4 @@
+# spelling::spell_check_package()
```

Comment:
I tried to include it into Linux builds, but somehow I could not see the output of the spellchecker.
So I added it to the windows build (appveyor.yml) instead